docs/aws-lambda-extension.asciidoc

```diff
@@ -98,13 +98,34 @@ For a _minimal configuration_ you need to specify the `ELASTIC_APM_LAMBDA_APM_SE
 [[aws-lambda-handler]]
 === Configuring the Agent and Lambda Function handler
 
-Once you've installed the extension, there's one last step to take. You'll need to wrap the Lambda function handler.
+Once you've installed the extension, there's one last step to take. You'll need to install the correct language agent and wrap the Lambda function handler.
```

Comment:
This feels like a bit of a lie as "one last step" is really six more steps. What about this instead?

```suggestion
After the extension has been installed, install the relevant language agent and wrap the Lambda function handler.
```